api-reference/beta/api/application_restore.md

```diff
@@ -2,7 +2,7 @@
 
 
 ## Prerequisites
```

Comment:
You need to remove (or change) this topic. You'd get the list of deleted application using
GET ../beta/directory/deletedItems/microsoft.graph.application

Restore would be through deletedItems
POST ../beta/directory/deletedItems/{id}/restore.